src/styles/blocks/header.scss

```diff
@@ -0,0 +1,23 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.is-active {
+  color: #00acdc;

```

Comment:
is these styles relates to header?
will be better to create separate modifier with these styles for the links, and use `is-active` class only in html for tests passing